src/modules/get-shows.js

```diff
@@ -0,0 +1,17 @@
+import { tvApi } from './api';
+import elementInfo from './element-info';
+
+// GET SHOWS FROM API https://www.tvmaze.com/api/shows
+const getShows = async () => {
+  const response = await fetch(`${tvApi}/show`);
+  const data = await response.json();
+  // we can display more or less shows if we like
+  const shows = data.slice(0, 20);
+  shows.forEach((data) => {
+    if (data.image !== null) {
+      elementInfo.renderCard(data.name, data.image.medium, data.id, data.summary);
+    }
+  });
```

Comment:
- I think its better to add the catch err method to the data you are fetching because it helps you to handle the error that comes with fetching API👌😀

- I notice you have some comments in line 4 and 8. Can you take them out so that you can have cleaner code base. Or what do you think?